notebooks/official/tabnet/tabnet_vertex_tutorial.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    LEARNER_IMAGE_URI = "gcr.io/sungyong-internship/tabnet:fix_header_csv"
</pre><p>Can we please switch to a production image location? This notebook is aiming to join the official location. Otherwise, we may need to look at community until the issue is resolved.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Yes, the image production will be available soon (likely this weekend).</p><p>I want to start the review process so that I can submit the PR right after the official image is available.</p>

Reply:
I update the prod image now.